cmd/multirootca/ca.go

```diff
@@ -92,6 +92,9 @@ func main() {
 	}
 
 	http.HandleFunc("/api/v1/cfssl/authsign", dispatchRequest)
+	http.HandleFunc("/api/v1/cfssl/healthcheck", func(w http.ResponseWriter, r *http.Request) {
+		w.WriteHeader(204)
```

Comment:
if only we had a [200 level version of this](https://http.cat/418) for "I'm a mint tea pot"

Reply:
yeah k8s requires 200-399 for this. I love me a teapot